website/docs/bottomsheet.md

```diff
@@ -42,9 +42,13 @@ const list = [
 
 ## Props
 
-- [`containerStyle`](#containerStyle)
-- [`isVisible`](#isvisible)
-- [`modalProps`](#modalprops)
+- [Usage](#usage)
+- [Props](#props)
+- [Reference](#reference)
+  - [`containerStyle`](#containerstyle)
+  - [`isVisible`](#isvisible)
+  - [`touchOutsideToClose`](#touchoutsidetoclose)
```

Comment:
This shouldn't be so specific. This should instead match the `Overlay` component's prop `onBackdropPress` so that people can hook in their own function instead of being locked into only a single action.

Reply:
Touching outside to close a bottom sheet is a very natural behavior. If someone has to hook this event, maybe another method could be added here just as you mentioned?

Reply:
No, I do not agree with maintaining 2 props when 1 would be just fine. Writing a hook to close the sheet is a single line and not burdensome. And any developer should be very familiar with basic `state` variables in order to achieve it through a generic hook.

Reply:
Plus `isVisible` is a prop not a part of the state. So this method just won't make sense/won't work.

Reply:
It was before I merged latest changes. BTW, I believe there would not be many develops who would like to hook this action in most cases and I saw several developers ask for this basic feature. It is different from `onBackdropPress`.